src/api/app/assets/stylesheets/webui2/list-group-item-integrated.scss

```diff
@@ -1,3 +1,7 @@
 .list-group .list-group-item-integrated:last-child {
     border-bottom: none;
-}
\ No newline at end of file
+}
+
+.no-list-style {
```

Comment:
There are [content classes](https://getbootstrap.com/docs/4.1/content/typography/#lists) for this.